server/models/Cart.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose');
```

Comment:
Hi Betty, I'm noticing a number of code style inconsistencies in these model files... inconsistent use of double vs single quotes, inconsistency of using spaces between the colon and value in objects, inconsistent spacing between code and closing curly braces, etc. These items should be fixable by your in-editor formatting.